deployments/upgrades/export_configs_to_helm.py

```
@@ -105,13 +128,20 @@ def fetch(base_url, path, headers):
         with urllib.request.urlopen(req, timeout=30) as resp:
             return json.loads(resp.read())
     except urllib.error.HTTPError as error:
-        print(f'Error fetching {path}: HTTP {error.code}', file=sys.stderr)
+        body = ''
         try:
             body = error.read().decode()
-            print(f'  {body[:200]}', file=sys.stderr)
-        except Exception as read_error:
-            print(f'  (Could not read error body: {read_error})',
+        except Exception:  # pylint: disable=broad-except
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)